[quant] Factored out common code in batchnorm.py

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43393 [quant] BatchNorm1d
* **#43392 [quant] Factored out common code in batchnorm.py**

Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

Commit:
[quant] Factored out common code in batchnorm.py

[ghstack-poisoned]

vkuzo reviewed (approved):
I don't feel strongly, this looks good but the previous code looked good as well

Commit:
Update on "[quant] Factored out common code in batchnorm.py"




[ghstack-poisoned]

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

z-a-f commented:
> I don't feel strongly, this looks good but the previous code looked good as well

I agree this is not necessary, but there is so much copy paste in the BN, especially with introduction of BN1d

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

Commit:
Update on "[quant] Factored out common code in batchnorm.py"


Differential Revision: [D23301022](https://our.internmc.facebook.com/intern/diff/D23301022)

[ghstack-poisoned]

z-a-f commented:
@zafartahirov has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D23301022).
